src/blocks/testimonials/block.json

```diff
@@ -0,0 +1,60 @@
+{
+	"name": "coblocks/testimonials",
+	"category": "layout",
+	"attributes": {
+		"columns": {
+			"type": "number",
+			"default": 2
+		},
+		"gutter": {
+			"type": "string",
+			"default": "medium"
+		},
+		"headingLevel": {
+			"type": "integer",
+			"default": 4
+		},
+		"showRoles": {
+			"type": "boolean",
+			"default": true
+		},
+		"showImages": {
+			"type": "boolean",
+			"default": false
+		},
+		"styleName": {
+			"type": "string",
+			"default": "boxy"
+		},
+		"backgroundColor": {
+			"type": "string"
+		},
+		"customBackgroundColor": {
+			"type": "string"
+		},
+		"textColor": {
+			"type": "string"
+		},
+		"customTextColor": {
+			"type": "string"
+		},
+		"bubbleBackgroundColor": {
+			"type": "string"
+		},
+		"bubbleTextColor": {
+			"type": "string"
+		},
+		"customBubbleTextColor": {
+			"type": "string"
+		},
+		"customBubbleBackgroundColor": {
+			"type": "string"
+		},
+		"fontSize": {
+			"type": "string"
+		},
+		"customFontSize": {
+			"type": "number"
+		}
```

Comment:
These attributes should be removed completely in favor of block supports which define these attributes for us and allow us to align with core.
https://github.com/godaddy-wordpress/coblocks/blob/d233cbc771f8a481ffc771a6434601c3bf742ea3/src/blocks/counter/block.json#L35-L45